committing.rst

```diff
@@ -222,96 +157,95 @@ Working with Git_
 .. seealso::
    :ref:`gitbootcamp`
 
-As a core developer, the ability to push changes to the official Python
-repositories means you have to be more careful with your workflow:
+As a core developer, you have the ability to push changes to the official
+Python repositories, so you need to be careful with your workflow:
 
-* You should not push new branches to the main repository.  You can still use
-  them in your fork that you use for development of patches; you can
-  also push these branches to a **separate** public repository that will be
-  dedicated to maintenance of the work before the work gets integrated in the
-  main repository.
+* **You should not push new branches to the main repository.**  You can
+  still use them in the fork that you use for the development of patches.
+  You can also push these branches to a separate public repository
+  for maintenance work before it is integrated into the main repository.
 
-* You should not commit directly into the ``master`` branch, or any of the
-  maintenance branches (currently ``3.9`` and ``3.8``).
-  You should commit against your own feature branch, and create a pull request.
+* **You should not commit directly into the** ``master`` **branch, or any of the maintenance branches (currently 3.9 and 3.8).**
```

Comment:
Having to update this doc every year just to change the maintenance branch numbers seems like wasted work when the list of maintenance branches is on the home page for the devguide itself at https://devguide.python.org/#status-of-python-branches.

```suggestion
* **You should not commit directly into the** ``master`` **branch, or any of the maintenance branches.**
```